flapp-webapp/src/routes.js

```diff
@@ -0,0 +1,16 @@
+import LandingPage from './components/landingNavigation/LandingPage.vue';
+import login from './components/landingNavigation/login.vue';
```

Comment:
Please name the component files consistently, with title case.

Reply:
Fixed in the next pull request